bridge/qb/shared/main.lua

```diff
@@ -1,15 +1,17 @@
-local starterItems = require 'config.server'.starterItems
 local qbShared = require 'shared.main'
 
----@deprecated use starterItems in config/server.lua
-qbShared.StarterItems = {}
```

Comment:
Keep the default for client table? Otherwise this would now be nil when previously was an empty table.

Reply:
Yeah, might be better to just move the starter items to shared since the client is going to get them anyway at start it's not a big deal for the client to know what they are